handlers/developers_handler.js

```diff
@@ -47,27 +14,110 @@ var DeveloperHandler = {
     this.client = redis.createClient({password: process.env.REDIS})
   },
   /**
+   * Set the response object to be in the format of a successful reponse
+   * @param {object} res - The response object
+   * @param {string} username - The given username
+   * @param {object} token - The given testing-token
+   */
+  setStandardResponse: function (res, username, token) {
+    res.status(200)
+    res.format({
+      json: function () {
+        res.json({
+          username,
+          token
+        })
+      }
+    })
+  },
+  /**
+   * Set the response object to be in the format of a successful reponse
+   * @param {object} res - The response object
+   * @param {string} developers - The set of usernames and tokens returned from the query
```

Comment:
a set of usernames is a `{String}`?  Are you sure it's not a `{String[]}` ? if it is just a string, are they comma-delimited or what?